src/domain/keyword/query.ts

```diff
@@ -54,3 +54,18 @@ export const QUERY_KEYWORD = gql`
     }
   }
 `;
+
+export const QUERY_POPULAR_KEYWORDS = gql`
+  query PopularKeywords($amount: Int, $showAllKeywords: Boolean) {
+    popularKultusKeywords(amount: $amount, showAllKeywords: $showAllKeywords) {
+      meta {
+        count
+        next
+        previous
+      }
```

Comment:
Just a note that it's always a single page / unpaginated result. API offers next and previous in meta only for consistency. They could be left out.